app/src/marketplace/types/project-types.ts

```diff
@@ -35,35 +46,34 @@ export type Project = {
     id: ProjectId | null | undefined
     name: string
     description: string
-    chain: string
     owner: string
     imageUrl: string | null | undefined
     newImageToUpload?: File | null | undefined
     thumbnailUrl: string | null | undefined
     state?: ProjectState
     created?: string
     updated?: string
-    category: CategoryId | null | undefined
     streams: StreamIdList
     previewStream: StreamId | null | undefined
     previewConfigJson?: string | null | undefined
     minimumSubscriptionInSeconds?: number
     ownerAddress: Address
-    beneficiaryAddress: Address
-    pricePerSecond: NumberString
-    priceCurrency: ContractCurrency
-    timeUnit?: TimeUnit | null | undefined
-    price?: NumberString
-    isFree?: boolean
-    type?: ProjectType
-    requiresWhitelist?: boolean
-    pendingChanges?: PendingChanges
+    type?: ProjectTypeEnum
     termsOfUse: TermsOfUse
     contact: ContactDetails | null | undefined
+    // Paid project
+    salePoints?: SalePoint[]
+    // Data union
+    chain?: string
     dataUnionDeployed?: boolean
-    pricingTokenAddress: Address
-    pricingTokenDecimals: number,
     existingDUAddress?: string
+    pricingTokenAddress?: Address
```

Comment:
Should we express these payment related things for Data Unions also as a SalePoint?

Reply:
Sounds good, but I'd name the field dataUnionSalePoint

Reply:
Does it need to be separate? It would make buying the project simpler as we could refer to only `salePoints` property instead of checking project type and using then different property `dataUnionSalePoint`.

Reply:
I thought about having this separate because this is not only a Sale Point, it also determines to which chain the DU is being deployed